MockKit/MockPumpManager.swift

```diff
@@ -619,6 +619,12 @@ public final class MockPumpManager: TestingPumpManager {
     }
 }
 
+extension MockPumpManager {
+    public func acceptDefaultsAndSkipOnboarding() {
+        // TODO: Implement This
+    }
```

Comment:
is there something to implement here? The comment may be misleading.

Reply:
see https://github.com/tidepool-org/LoopKit/pull/489/files#r1156231008